chart/templates/workers/worker-deployment.yaml

```diff
@@ -219,6 +219,9 @@ spec:
         {{- if and (.Values.dags.gitSync.enabled) (not .Values.dags.persistence.enabled) }}
         {{- include "git_sync_container" . | indent 8 }}
         {{- end }}
+{{- if .Values.workers.extraEnv }}
+{{ tpl (toYaml .Values.workers.extraEnv) $ | indent 12 }}
```

Comment:
Why no waitForMigrations here? Don't forget the k8s pod_template_file too.

Reply:
@jedcunningham There isn't currently a waitForMigrations property for worker pods. I have added the same set of envs used for the worker container for now. Or I could introduce a new property to add custom envs for the migrations container.
Could you direct me on how to overwrite the base pod_template_file with labels and envs?

Reply:
Here is the pod_template_file: https://github.com/apache/airflow/blob/main/chart/files/pod-template-file.kubernetes-helm-yaml

Reply:
Thanks. Have pushed changes to the pod template file